### PR TITLE
Issue #2908887: Move to Paragraphs experimental Widget

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [x] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [x] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [x] Necessary update hooks are provided.
- [x] User roles have correct access for new introduced permission.
- [x] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [x] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by dubcanada** [comment]
What's the status of this? I'd like to use the experimental widget. Does anything need to be finished/tested?

**Review by dbosen — CHANGES_REQUESTED** [review]
~~The option to confirm deletion of paragraphs should be enabled by default for both the article and the term paragraphs field.~~

**Comment by mtodor** [comment]
~~We should add testes for - delete confirmation and single option to button features on paragraphs. That could be in one test, both tests are related to remove action.~~

File: modules/thunder_updater/updates.yml
Comment (mtodor) [inline modules/thunder_updater/updates.yml]: This should be adjusted, because version has changed.

File: scripts/travis/04-install-thunder.sh
Comment (mtodor) [inline scripts/travis/04-install-thunder.sh]: ~~Need to be adjusted before merge and we need release of theme first anyway.~~

File: tests/src/FunctionalJavascript/ThunderJavascriptTestBase.php
Comment (mtodor) [inline tests/src/FunctionalJavascript/ThunderJavascriptTestBase.php:122]: ~~Should be documented in developer documentation.~~

**Review by mtodor — CHANGES_REQUESTED** [review]

File: composer.json
Comment (dbosen) [inline composer.json]: we need to wait for paragraphs features release, and change this line before merge

File: composer.json
Comment (dbosen) [inline composer.json]: ~~Has to be reverted before merge~~

File: thunder.post_update.php
Comment (dbosen) [inline thunder.post_update.php]: Module installation should be done in update hook

File: thunder.install
Comment (dbosen) [inline thunder.install]: This check leads to the problem, that if either default_content or better_normalizers is missing, the whole update fails. Since we would still be able to do most of the updates, this should be changed.
Proposal: 
Split update hook into multiple update hooks, each of them with different dependencies that define if the update was successful. This way we can do most updates, that for example only need the paragraphs feature modules, and let only the update fail, that actually needs default_content or better_normalizers. Also, the Liveblog update could be separated out and only done when needed.
The general idea is to separate those updates that have independent requirements.

**Comment by chrfritsch** [comment]
~~So I am testing things a bit. This one might be out of scope, but I wanted to note it anyway :-)~~

~~If the "Autocollapse" feature is enabled and you change something in an image paragraph, then switch to a text pragraph, your changes are lost :-(~~

**Comment by chrfritsch** [comment]
~~The "Add above" functionality doesn't work properly when "Add in-between" is enabled. I tested that in Seven theme, because "Add above" doesn't work in "Thunder admin" currently.~~

**Review by mtodor — APPROVED** [review]

**Review by dbosen — APPROVED** [review]

**Pull request merged**
